### DISCUSSION TITLE
Made some change in packages and i want to publish it

### DISCUSSION THREAD

**Discussion by GHOST117s** [Z3d0X/filament-logger#65] [Q&A] [answered]
I have made some changes to the packages
-added delete option 
-bulk delete 
and some changes in resource 

so can I publish that

so that anyone who pulls my repo has that changes

**Comment by Z3d0X** [comment, marked as answer]
I don't think logs should be deleted this way. Sure you could be able to clear out old logs (`spatie/laravel-activitylog` already has this function), but i don't think you should be able to take an action & just delete the log of it..
